tests/test_pagination.py

```diff
@@ -0,0 +1,52 @@
+from tap_tester import runner, connections
+
+from base import TestLinkedinAdsBase
+
+class LinkedinAdsPaginationTest(TestLinkedinAdsBase):
+
+    @staticmethod
+    def name():
+        return "tap_tester_linkedin_ads_pagination_test"
+
+    def test_run(self):
+        # page size for "campaign_groups" set in the tap
+        page_size = 100
+        conn_id = connections.ensure_connection(self)
+
+        # Checking pagination for "campaign_groups" stream
+        expected_streams = ["campaign_groups"]
```

Comment:
The assumption that if one stream can paginate, all can streams can is an anti-pattern. We should only omit streams from this test if the barrier to generate sufficient data is exceedingly high.

Reply:
@kspeer825 Added page size as 1 for all the streams available for pagination in the pagination test. Please refer [Doc](https://jira.talendforge.org/browse/TDL-13963?focusedCommentId=950284&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-950284) for more details.

Reply:
If we are going to override the page size I would think the absolute minimum value used should be 2. Paging after only 1 record does not seem like it would necessarily mimic the same code path that standard pagination would. But I am not familiar with the implementation, so it may be ok.

Reply:
@kspeer825, setting `page_size` to 1 works. I validated that the tap is correctly making API requests with page size 1 by looking at the CircleCi logs.